export-or-backup-using-dumpling.md

```diff
@@ -10,71 +10,120 @@ category: how-to
 
+Dumpling 的更多具体用法可以使用 --help 指令查看，或者查看[中文使用手册](https://github.com/pingcap/dumpling/blob/master/docs/cn/user-guide.md)。
+
 使用 Dumpling 时，需要在已经启动的集群上执行导出命令。本文假设在 `127.0.0.1:4000` 有一个 TiDB 实例，并且这个 TiDB 实例中有无密码的 root 用户。 
 
+## 下载地址
+
+最新版 Dumpling 的下载地址见[下载链接](https://download.pingcap.org/dumpling-nightly-linux-amd64.tar.gz)。
+
 ## 从 TiDB 导出数据
 
-使用如下命令导出数据：
+### 导出到 sql 文件
+
+Dumpling 默认导出数据格式为 sql 文件。也可以通过设置 --filetype 为 sql 导出数据到 sql 文件：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 dumpling \
   -u root \
   -P 4000 \
-  -H 127.0.0.1 \
+  -h 127.0.0.1 \
   --filetype sql \
   --threads 32 \
   -o /tmp/test \
-  -F $(( 1024 * 1024 * 256 ))
+  -F 256
 ```
 
-上述命令中，`-H`、`-P`、`-u` 分别是地址，端口，用户。如果需要密码验证，可以用 `-p $YOUR_SECRET_PASSWORD` 传给 Dumpling。
+上述命令中，`-h`、`-P`、`-u` 分别是地址，端口，用户。如果需要密码验证，可以用 `-p $YOUR_SECRET_PASSWORD` 传给 Dumpling。
 
-默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。假如导出数据的格式是 CSV（使用 `--filetype csv` 即可导出 CSV 文件），还可以使用 `--sql <SQL>` 导出指定 SQL 选择出来的记录，例如，导出 `test.sbtest1` 中所有 `id < 100` 的记录：
+### 导出到 csv 文件
+
+假如导出数据的格式是 CSV（使用 `--filetype csv` 即可导出 CSV 文件），还可以使用 `--sql <SQL>` 导出指定 SQL 选择出来的记录，例如，导出 `test.sbtest1` 中所有 `id < 100` 的记录：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 ./dumpling \
   -u root \
   -P 4000 \
-  -H 127.0.0.1 \
+  -h 127.0.0.1 \
   -o /tmp/test \
   --filetype csv \
   --sql "select * from `test`.`sbtest1` where id < 100"
 ```
 
-注意，`--sql` 选项暂时仅仅可用于导出 csv 的场景。但是仍旧可以用 `--where` 来过滤要导出的行，使用以下指令，可以导出所有 `id < 100` 的记录：
-
 > **注意：**
 > 
-> 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
+> 1. `--sql` 选项暂时仅仅可用于导出 csv 的场景。
+>
+> 2. 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
+
+### 筛选导出的数据
+
+#### 使用 `--where` 指令筛选数据
+
+默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 ./dumpling \
   -u root \
   -P 4000 \
-  -H 127.0.0.1 \
+  -h 127.0.0.1 \
   -o /tmp/test \
   --where "id < 100"
 ```
 
-> **注意：**
-> 
-> 目前 Dumpling 不支持仅导出用户指定的某几张表（即 `-T` 标志，见[这个 issue](https://github.com/pingcap/dumpling/issues/76)）。如果你确实需要这些功能，可以先使用 [MyDumper](/backup-and-restore-using-mydumper-lightning.md)。
+上述命令将会导出各个表的 id < 100 的数据。
+
+#### 使用 `--filter` 指令筛选数据
+
+Dumpling 可以通过 --filter 指定 table-filter 来筛选特定的库表。table-filter 的语法与 .gitignore 相似，[详细语法参考](https://github.com/pingcap/tidb-tools/blob/master/pkg/table-filter/README.md)。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+./dumpling \
+  -u root \
+  -P 4000 \
+  -h 127.0.0.1 \
+  -o /tmp/test \
+  --filter "employees.*"
+  --filter "*.WorkOrder"
+```
+
+上述命令将会导出 `employees` 数据库的所有表，以及所有数据库中的 `WorkOrder` 表。
+
+#### 使用 `-B`，`-T` 指令筛选数据
+
+与 Mydumper 一致，Dumpling 也可以通过 -B 或 -T 参数导出特定的数据库/表。
+
+例如通过指定:
+
+- `-B employees` 导出 `employees` 数据库
+- `-T employees.WorkOrder` 导出 `employees.WorkOrder` 数据表
```

Comment:
please mention that `-T` only accepts qualified table name, i.e. `-T WorkOrder` is an error.

also mention that `-T` and `-f` cannot be used together.